tests/t282_agent_depth.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+
+import subprocess as sp
+from time import sleep
+
+from runtest import TestBase
+
+class TestCase(TestBase):
+    """Test changing the tracing depth in the target from the agent.
+
+    The target itself has an original depth limit of 4, and a depth trigger on
+    the 'trigger' function. We decrease the depth to 2 and then increase the
```

Comment:
I don't think we have the trigger support yet.

Reply:
I was referring to static triggers, that are defined at start-up. But I'm removing that.